doc/explanation/containers_and_vms.md

```diff
@@ -0,0 +1,24 @@
+(containers-and-vms)=
+# About containers and VMs
+
+LXD provides support for two different types of {ref}`instances <expl-instances>`: *system containers* and *virtual machines*.
+
+When running a system container, LXD simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
+
+When running a virtual machine, LXD uses the hardware of the host system, but the kernel is provided by the virtual machine. Therefore, virtual machines can be used to run, for example, a different operating system.
+
+## Application containers vs. system containers
+
+Application containers (as provided by, for example, Docker or Kubernetes) package a single process or application. System containers, on the other hand, simulate a full operating system and let you run multiple processes at the same time.
```

Comment:
Kubernetes doesn't provide application containers, it just orchestrates them. I'd not mention k8s here.

Reply:
Removed. :)